content/providers/provider-azure/authentication.md

```diff
@@ -0,0 +1,154 @@
+* [User-assigned managed identity](https://learn.microsoft.com/en-us/azure/aks/use-managed-identity#bring-your-own-managed-identity)
+
+## Service principal with Kubernetes secret
+
+A service principal is an application within the Azure Active Directory that
+passes `client_id`, `client_secret`, and `tenant_id` authentication
+tokens to create and manage Azure resources. 
+
+### Create a service principal using the Azure CLI tool
+
+First, find the Subscription ID for your Azure account.
+
+```shell
+az account list
+```
+
+Note the value of the `id` in the return output. 
+
+Next, create a service principle `Owner` role. Update the `<subscription_id>`
+with the `id` from the previous command.
+
+```shell
+az ad sp create-for-rbac --sdk-auth --role Owner --scopes /subscriptions/<subscription_id> \
+  > azure.json
+```
+
+The `azure.json` file in the preceding command contains the client ID, secret, and
+tenant ID of your subscription.
+
+
+Next, use `kubectl` to associate your Azure credentials file with a generic
+Kubernetes secret.
+
+```shell
+kubectl create secret generic azure-secret -n upbound-system --from-file=creds=./azure.json
+```
+
+## Configure your provider
+
+Apply these changes to your `ProviderConfig` file. 
+
+```yaml {label="secretPC", copy-lines="5-11"}
+apiVersion: azure.upbound.io/v1beta1
+metadata:
+  name: default
+kind: ProviderConfig
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: upbound-system
+      name: azure-secret
+      key: creds
+```
+
+Your credential `source` must be `Secret` and you must specify the namespace,
+name, and key if you used different values.
+
+Apply your configuration.
+
+## System-assigned managed identity
+
+The system-assigned managed identity allows you to associate the provider with
+your
+AKS cluster automatically without manually
+managing credentials.
+
+### Create a system-assigned managed identity
+
+A system-assigned managed identity is automatically created when you create
+an AKS cluster. This section covers creating a new identity in a new cluster.
+
+Create a resource group.
+
+```shell
+az group create --name myResourceGroup --location westus2
+```
+
+Create an AKS cluster with the `--enable-managed-identity` flag.
+
+```shell
+az aks create -g myResourceGroup -n myManagedCluster --enable-managed-identity
+```
+
+Use the `aks get-credentials` command to generate the kubeconfig file
+for your AKS cluster. This file contains the authentication and connection
+information for your cluster.
+
+```shell
+az aks get-credentials --resource-group myResourceGroup --name myManagedCluster
+```
+
+To switch from a service principal to a system-assigned managed identity,
+use the `aks update` command.
+
+```shell
+az aks update -g myResourceGroup -n myManagedCluster --enable-managed-identity
+```
+
+### Configure your provider
```

Comment:
Does Crossplane have to run inside the same AKS cluster? For example, can I use this auth method with Kind on my laptop? 
Running Crossplane in the same managed cluster is a requirement for AWS but I have no idea if it's true for Azure

Reply:
I think so because of this line in [the AUTHENTICATION.md](https://github.com/upbound/provider-azure/blob/main/AUTHENTICATION.md) doc.

```
One important point to note here is that the AKS cluster's kubelet identity must have the necessary role assignments [[3](https://learn.microsoft.com/en-us/azure/aks/use-managed-identity)] so that the cloud API calls made by provider-azure will be authorized. The set of roles required by provider-azure depends on the resources being managed by it.
```

And with the User-assigned managed identity:

```
This method is similar to authentication using system-assigned managed identities and requires that the AKS cluster is using a user-assigned managed identity as its kubelet identity [[3](https://learn.microsoft.com/en-us/azure/aks/use-managed-identity)]. An example ProviderConfig is as follows:
```

Reply:
cool. Let's make sure there's a mention of that requirement